src/components/BookDetail/BookDetail.jsx

```diff
@@ -3,6 +3,7 @@ import { useParams } from "react-router-dom"
 import Reviews from "./Reviews/Reviews";
```

Comment:
Please remove semi colon from end of line 3 and 6 and 10 if it is not functional

Reply:
Done, Semi-colons removed.